indico_jacow/plugin.py

```diff
@@ -36,9 +41,28 @@ class JACOWPlugin(IndicoPlugin):
     def init(self):
         super().init()
         self.template_hook('abstract-list-options', self.inject_export_button)
+        self.connect(signals.event.sidemenu, self.extend_event_menu)
+        self.connect(signals.menu.items, self.add_sidemenu_item, sender='event-management-sidemenu')
 
     def inject_export_button(self, event=None):
         return render_plugin_template('export_button.html', event=event)
 
+    def extend_event_menu(self, sender, **kwargs):
+        def _statistics_visible(event):
+            if not session.user or not event.has_feature('abstracts'):
+                return False
+            return any(track.can_review_abstracts(session.user) for track in event.tracks)
+            return has_user_tracks(event, session.user)
+
+        return MenuEntryData(title=_('My Statistics'), name='abstract_reviewing_stats',
+                             endpoint='jacow.reviewer_stats', position=1, parent='call_for_abstracts',
```

Comment:
have you tested this and confirmed that it works? since the endpoint is in a plugin and we do not use `url_for_plugin` to build it, you may need `plugin_jacow.reviewer_status`..

Reply:
nevermind, just spotted the magic we have to handle it